numpy/linalg/umath_linalg.c.src

```diff
@@ -1092,7 +1092,10 @@ static inline void
     fortran_int info = 0;
     int i;
     /* note: done in place */
-    LAPACK(@cblas_type@getrf)(&m, &m, (void *)src, &m, pivots, &info);
+    if (m != 0) {
+        /* Lapack (at least some versions) do not handle n == 0 */
+        LAPACK(@cblas_type@getrf)(&m, &m, (void *)src, &m, pivots, &info);
```

Comment:
Dunno, my guess was I had problems on my system, or, I just did not know if it was necessary or not. The `max(1, m)` thing may just be the reason too I suppose.